api-report/map.api.md

```diff
@@ -65,6 +66,8 @@ export interface IDirectoryDataObject {
 export interface IDirectoryEvents extends IEvent {
     // (undocumented)
     (event: "containedValueChanged", listener: (changed: IValueChanged, local: boolean, target: IEventThisPlaceHolder) => void): any;
+    // (undocumented)
+    (event: "disposed", listener: (target: IEventThisPlaceHolder, error?: Error) => void): any;
```

Comment:
Fo we ever have an error object here? If not, feels like argument should not be here.

Reply:
not used today but left for future. Removing it for now as could be added later as it is not getting used currently.